src/pb_tensor.cc

```diff
@@ -31,6 +31,7 @@
 #ifdef TRITON_PB_STUB
 #include "pb_stub_utils.h"
 namespace py = pybind11;
+using namespace py::literals;
```

Comment:
I would rather not use `using namespace` since it can pollute the symbol space and create conflicts. You can create aliases for easier usability. For example, `namespace pyl = py::literals`.